IdokladSdk.IntegrationTests/Tests/Clients/IssuedDocumentPayment/IssuedDocumentPaymentTests.Async.cs

```diff
@@ -11,7 +11,7 @@ public partial class IssuedDocumentPaymentTests
         public async Task List_SuccessfullyGetAsync()
         {
             // Act
-            var data = (await _client.List().GetAsync()).AssertResult();
+            var data = (await _client.List().Sort(x => x.DateOfPayment.Asc()).GetAsync()).AssertResult();
```

Comment:
Proč se to sortuje, když se pak očekává jen jeden výsledek?

Reply:
chcel som sa iba uistiť, že ten sort API akceptuje, nechcel som vytvárať nový test a ani data